packages/test-utils/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@fuel-ts/test-utils",
```

Comment:
I [wasn't able](https://github.com/FuelLabs/fuels-ts/pull/1356/files#r1362324925) to export `TestNodeLauncher` via `fuels/test-utilities` so I created this package because I think we should have one package that serves as an "add this package to your devDependencies if you want to do anything testing-related with our sdk". I don't have a hard opinion on this; we can recommend the users to add `@fuel-ts/contract` and import `TestNodeLauncher` from there as well, although I think having a dedicated "test-utils" package makes sense.